examples/simple_state.rs

```diff
@@ -56,6 +56,11 @@ fn current_count_render(
                         text: "Click me!".into(),
                         ..Default::default()
                     }}
+                    styles={KStyle {
+                        font_size: StyleProp::Value(48.),
```

Comment:
We can use `into` for a lot of these instead of wrapping everything in `StyleProp:Value`.